packages/date-parser/src/parsers/yearMonth.js

```diff
@@ -0,0 +1,71 @@
+import Chrono from 'chrono-node';
+import momentFromStruct from '../helpers/momentFromStruct';
+import chronoDateStructFromMoment from '../helpers/chronoDateStructFromMoment';
+
+const parser = new Chrono.Parser();
+
+parser.pattern = () => {
+  /*
+    Date format with slash "/" between numbers like ENSlashDateFormatParser,
+    but this parser expect year before month.
+    - YYYY/MM
+    - YYYY-MM
+  */
+
+  const delimiter = '[\\/-]';
+  const year = '([0-9]{4})';
+  const month = '(?:([0-9]{1,2}))';
+  const endOfWord = '(?=\\W|$)';
```

Comment:
P2: clarify, not sure about this regex, can u help explain?

Reply:
or can add tests to show why we need this part